scripts/hooks/enforce_boilerplate.py

```diff
@@ -17,6 +17,8 @@
 from typing import NoReturn
 
 BOILERPLATE = [
```

Comment:
This code will need to be present at the bottom of every test, otherwise pre-commit checks will fail.

Reply:
Actually let's just remove this for now. I think we're going to end up seeding `np.random` and python `random` automatically on `import legate`.